src/dune_rules/module_compilation.ml

```diff
@@ -309,15 +309,17 @@ let build_module ?(force_write_cmi = false) ?(precompiled_cmi = false) cctx m =
           let sctx = CC.super_context cctx in
           let dir = CC.dir cctx in
           let action_with_targets =
-            Jsoo_rules.build_cm
-              sctx
-              ~dir
-              ~in_context
-              ~src:(Path.build src)
-              ~obj_dir
-              ~config:None
+            List.map Jsoo_rules.Config.all ~f:(fun config ->
```

Comment:
@rgrinberg, it'd be better if we could construct rules dynamically based on path (like done for ppxes executables) instead of relying on a restricted list of variants  (e.g. `Jsoo_rules.Config.all`). I think you told me in the past that this feature was planned for later. Was there any progress made on that front ?

Reply:
ping @rgrinberg

Reply:
The feature still hasn't yet made it in. But you could copy the trick that we use for ppx.